### PR TITLE
Niupadhy debug cls Adaptor changes,

### PR DESCRIPTION
**Pull request by nishantupadhyay123**
testing done for 
CLS , not debug
cls , debug
debug , not cls
not cls , not debug 

All other collector and analyzer functionality and CLS test update is retained and tested.

### CODE REVIEW

File: cafy_pytest/plugin.py [hunk 1]
```diff
@@ -515,49 +515,63 @@ def pytest_configure(config):
                 del os.environ["hybrid_mode"]
 
         #Debug Registration Server code
-        if CLS and int(CLS):
-            registration_id = os.environ.get("REG_ID", None)
-            CafyLog.registration_id = registration_id 
+        #Cant introduce switch case as this plugin is common for 3.6 and 3.11 env.
+        call_registeration = False
+        if CLS and not cafykit_debug_enable :
```
Comment (Wendong-Wu) [inline cafy_pytest/plugin.py:520]: If we change the behavior here, we must change the CLS definition in line 58 to `cd /auto/tftp-niupadhy-sjc/cls/311_env/bin/`

File: cafy_pytest/plugin.py [hunk 2]
```diff
@@ -515,49 +515,63 @@ def pytest_configure(config):
...
+            call_registeration = False
+            reg_id = os.environ.get("REG_ID", None)
+            CafyLog.logstash_server = LOGSTASH_SERVER
+            CafyLog.logstash_port = int(LOGSTASH_PORT)
```
Comment (gurprand) [inline cafy_pytest/plugin.py:524]: if LOGSTASH_PORT is not None: need to be checked

Comment (nishantupadhyay123) [reply cafy_pytest/plugin.py:530]: The check added at the top to set CLS= 0 when the logstahs and etc is None.
These are env variables so they need to convert from string to typical type aka integer.

Comment (nishantupadhyay123) [reply cafy_pytest/plugin.py:530]: Please re-open if the resolve is not supported.

File: cafy_pytest/plugin.py (hunk 1 above)
Comment (nishantupadhyay123) [reply cafy_pytest/plugin.py:526]: Did the change at line 58 and added comments how to improve it next iteration.

**Review by gurprand — APPROVED** [review]

**Pull request merged**